### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pkalapat**
Added details in README

### CODE REVIEW

**Review by shmcfarl — COMMENTED** [review]
LGTM

**Review by naor-cisco — COMMENTED** [review]
@pkalapat I must reject this ReadMe change
It looks like you encountered "level 0" issues ( for example "On a Mac, you can resolve this issue by going to Settings > Security & Privacy > Privacy > Developer Tools and checking the box next to "functionclarity" as shown here" ) an put it in ReadMe. From my point of view, a change will be needed if there is a mistake or something that is not clear.

I will suggest to open "beginners guide" and set those instruction overthere.

File: README.md [hunk 1]
```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
...
+
+![image](https://user-images.githubusercontent.com/3701244/207755079-688c7a83-6e7b-4c5c-ba06-33c8fbe4aea8.png)
+
+Go into Finder and locate the `functionclarity` binary and right-click on the file and click "open". Another popup will appear that looks like this:
+
+![image](https://user-images.githubusercontent.com/3701244/207755405-9d784f33-f046-4955-bacc-fb4acc54f251.png)
+
+A new terminal window will open and the command will run one time. You can close this terminal window if you use another terminal type like iTerm2 or you can leave it open for the additional FunctionClarity command-line steps below.
+
+## AWS IAM Role / Policy Creation
```
Comment (naor-cisco) [inline README.md:56]: @pkalapat what you intended to do overhere?

File: README.md [hunk 2]
```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
...
+function-clarity function code upload successfully
+deployment request sent to provider
+waiting for deployment to complete
+deployment finished successfully
+```
+
+## Use FunctionClarity to Sign Your Code
 
+```
+./functionclarity sign aws code src/
```
Comment (naor-cisco) [inline README.md:162]: in this line you instruct the user to sign the src code of what?

**Comment by shmcfarl** [comment]
> 



> @pkalapat I must reject this ReadMe change It looks like you encountered "level 0" issues ( for example "On a Mac, you can resolve this issue by going to Settings > Security & Privacy > Privacy > Developer Tools and checking the box next to "functionclarity" as shown here" ) an put it in ReadMe. From my point of view, a change will be needed if there is a mistake or something that is not clear.
> 
> I will suggest to open "beginners guide" and set those instruction overthere.

@naor-cisco, your reply didn't make much sense to me. Are you saying that this is not a real issue? It happens 100% of the time when you download the functionclarity binary (or any unsigned binary) on a Mac. Your guidance didn't make much sense either. Are you saying to create a new document ("beginners guide") and if so, how is that guide different than the purpose of this readme? Isn't it a beginners guide? :-)

File: README.md (hunk 1 above)
Comment (shmcfarl) [reply README.md:56]: The intention is that when you do the very first approval of the functionclarity binary run, it opens in the default terminal on your machine. That terminal may not be the one you typically use. It is, again, a way to ease the friction for interacting with a new project and an untrusted/unsigned binary (like the Mac approval steps above this section). If you believe that people don't need help with the consumption of your unsigned binary, then we can consider removing this information or stuffing it in a 'reference' section.

Comment (shmcfarl) [reply README.md:56]: Rule #1 of easing consumption of a new project, never assume anyone knows anything about using your project. :-)

File: README.md [hunk 3]
```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
...
+```
+
+Example output:
+```
+# ./functionclarity sign aws code src/
```
Comment (naor-cisco) [inline README.md:167]: again - in this example you are showing how to sign the function clarity code and not the example function.

File: README.md [hunk 4]
```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
...
+using config file: /Users/<ID>/.fc
```
Comment (naor-cisco) [inline README.md:168]: If you are showing an example - I guess that you can keep the "/Users/john/.fc"

File: README.md (hunk 1 above)
Comment (naor-cisco) [reply README.md:56]: I have to say... I don't agree with you about that....
If you are going to try an opensource solution - it means that you have an idea about how to run things on your comupter and what your capabilities are. when you get into an opensource that handles servereless functions - we assume that you are familliare with serverless frameworks and you handle serverless functions on you their day to day.

Yes - there is improvements that needs to be added to the ReadMe.  Yes we need to place somewhere what is the minimum permissions that you should have in order to run the solution. But I don't think that a specific instructions for Mac is something that belongs to the ReadMe. maybe FAQs? 


For example take a look at github cli (and you can find many more) and their instruction https://github.com/cli/cli/blob/trunk/docs/install_linux.md. The first paragraph leads you to their release page ( like we have ) and you can try to download their binary. Yes you will need to go through the same procedure that @pkalapat mentioned - but they are not specifing that.

File: README.md [hunk 5]
```diff
@@ -22,79 +22,199 @@ This version supports serverless functions on AWS (Lambda functions) only, suppo
 If a function is tagged as blocked, it will be prevented from being run by AWS when it is invoked.
 
 ## Download FunctionClarity
-Go to the [function clarity latest release](https://github.com/openclarity/functionclarity/releases/latest):
-* Create a folder and download:
-  * ```aws_function.tar.gz``` and extract it to the folder
-  * ```functionclarity-v<version_number>``` for your OS type and extract it to the folder 
-
-## Quick start
```
Comment (naor-cisco) [inline README.md:30]: bring this back

**Comment by shmcfarl** [comment]
Per our earlier discussion, we are closing this PR.

**Pull request closed without merging**